src/app/lib/logger.const.js

```diff
@@ -1,6 +1,16 @@
 const logCodes = {
   NO_TRANSLATION_FOUND: 'no_translation_found',
   NO_MEDIA_BLOCK: 'no_media_block',
+  DATA_NOT_FOUND: 'data_response_404',
+  SERVER_LISTEN_ERROR: 'server_listen_error',
+  SERVICE_WORKER_SENDFILE_ERROR: 'server_sendfile_error_sw',
+  MANIFEST_SENDFILE_ERROR: 'server_sendfile_error_manifest',
+  SERVER_SIDE_RENDER_REQUEST_RECEIVED: 'ssr_request_received',
+  SERVER_SIDE_REQUEST_FAILED: 'ssr_request_failed',
+  LOCAL_SENDFILE_ERROR: 'local_sendfile_error',
+  DATA_REQUEST_RECEIVED: 'data_request_received',
+  DATA_FETCH_ERROR: 'data_fetch_error',
+  ROUTING_INFORMATION: 'routing_info',
```

Comment:
It's great to have these here, defined as constants, since we're using them for monitoring & we'd like to use them for dashboards. Can we add a comment to note that if we update these values, we should ensure associated monitoring is updated too? 
Also a nitpick - can we please just make this an alphabetical list?

Reply:
Thanks for all the updates. I think my only outstanding one is to add a comment above here to note that if any of these strings are updated, that we should ensure monitoring dashboards & run books that use these for queries should be updated too.

Reply:
Done, thanks for reminding me! I also put a link to this file in the Sumologic shared Simorgh queries list - just in case!